src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -84,7 +84,7 @@ public void should_attach_exception_record_to_the_existing_case_with_no_evidence
 
         //then
         await("Exception record is attached to the case")
-            .atMost(60, TimeUnit.SECONDS)
+            .atMost(1200, TimeUnit.SECONDS)
```

Comment:
20 minutes??

Reply:
sorry Just for debugging purpose i increased timeout,i just change the title of the PR with WIP